modules/ROOT/pages/relational-database-connections-JDBC.adoc

```diff
@@ -61,7 +61,7 @@ If you use Maven to build your application, you can download and deploy the JDBC
 
 If you use Gradle to build your application, you can add a `copy` task to your `build.gradle` file. Then, add a dependency for that task to the `deploy` task in the `build.gradle` file so that the `copy` task runs after the server is created but before the application is deployed:
 
-[source,javascript]
+[source,groovy]
 ----
 configurations {
 	jdbcLib
```

Comment:
nit: reduce indentation of this line and the `jdbcLib` line below to match the new 2-space indent.

Reply:
fixed

Reply:
Line 70 still has a tab character as the indent, rather than 2 spaces https://github.com/OpenLiberty/docs/pull/4469/files#diff-daefca13d21b4cf1a6e401503fc4d80b065b021efc10615cbcbde1af3ca77dd4R70

Reply:
thanks- hard to tell from Atom